packages/remix-dev/config.ts

```diff
@@ -32,12 +32,12 @@ export type ServerBuildTarget =
 export type ServerModuleFormat = "esm" | "cjs";
 export type ServerPlatform = "node" | "neutral";
 
-type Dev = {
+interface Dev {
   port?: number;
   appServerPort?: number;
   remixRequestHandlerPath?: string;
   rebuildPollIntervalMs?: number;
-};
+}
```

Comment:
Can we keep this as a `type` rather than an `interface` so that it can't be accidentally extended?

`interface`s used to be preferred since they were more performant, but that stopped being the case a while ago 😅 so now I think it's nice to prefer `type` and only use `interface` if you need it